common/beerocks/bwl/dummy/base_wlan_hal_dummy.cpp

```diff
@@ -279,6 +279,12 @@ bool base_wlan_hal_dummy::dummy_send_cmd(const std::string &cmd) { return false;
 
 bool base_wlan_hal_dummy::dummy_send_cmd(const std::string &cmd, char **reply) { return false; }
 
+bool base_wlan_hal_dummy::process_nl_events()
+{
+    LOG(ERROR) << __func__ << "not implemented";
```

Comment:
Didn't we agree to remove the `__func__` from all logs?
#665

Reply:
first time i hear about it. why?

Reply:
It is redundant - see @vitalybu 's comments